.travis.yml

```diff
@@ -3,5 +3,4 @@ rust:
   - stable
   - beta
   - nightly
-  - 1.22.0
```

Comment:
Heh? You're just discarding our MSRV? That's a clear NACK. If this is accidental, please change the PR. I'd be ok with bumping the MSRV to 1.29 here just like we did in rust-bitcoin..